ValveResourceFormat/ClosedCaptions/ClosedCaptions.cs

```diff
@@ -118,7 +118,6 @@ public override string ToString()
             KVSerializer.Create(KVSerializationFormat.KeyValues1Text).Serialize(ms, captionsToExport, FileName);
 
             var sb = new StringBuilder();
-            sb.AppendLine(CultureInfo.InvariantCulture, $"// {Utils.StringToken.VRF_GENERATOR}");
```

Comment:
Why remove this?

Reply:
Valve's captioncompiler doesn't understand comments and fails to compile if it's included

Reply:
Are we even exporting in the format their compiler understands? Because i just threw it into kv1 for fun.

Reply:
Yes, it compiles and the tokens do get written into the file surprisingly.

Reply:
That's a funny coincidence.